Compile error with Qt 5.15.0: 'QPainterPath path' has initializer but incomplete type

Hi, it does fails to compile with the latest Qt on Archlinux. 
Relevant log:
```
[ 37%] Building CXX object CMakeFiles/DSView.dir/pv/toolbars/samplingbar.cpp.o
[ 37%] Building CXX object CMakeFiles/DSView.dir/pv/view/viewport.cpp.o
[ 38%] Building CXX object CMakeFiles/DSView.dir/pv/view/view.cpp.o
[ 38%] Building CXX object CMakeFiles/DSView.dir/pv/view/timemarker.cpp.o
/home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.cpp: In member function 'virtual void pv::view::Viewport::wheelEvent(QWheelEvent*)':
/home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.cpp:1009:38: warning: 'int QWheelEvent::delta() const' is deprecated: Use angleDelta() [-Wdeprecated-declarations]
 1009 |                 t->zoom(event->delta() / 80, event->x());
      |                                      ^
In file included from /usr/include/qt/QtGui/QNativeGestureEvent:1,
                 from /home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.h:35,
                 from /home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.cpp:23:
/usr/include/qt/QtGui/qevent.h:219:16: note: declared here
  219 |     inline int delta() const  { return qt4D; }
      |                ^~~~~
/home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.cpp:1009:55: warning: 'int QWheelEvent::x() const' is deprecated: Use position() [-Wdeprecated-declarations]
 1009 |                 t->zoom(event->delta() / 80, event->x());
      |                                                       ^
In file included from /usr/include/qt/QtGui/QNativeGestureEvent:1,
                 from /home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.h:35,
                 from /home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.cpp:23:
/usr/include/qt/QtGui/qevent.h:229:16: note: declared here
  229 |     inline int x() const { return int(p.x()); }
      |                ^
/home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.cpp:1014:32: warning: 'Qt::Orientation QWheelEvent::orientation() const' is deprecated: Use angleDelta() [-Wdeprecated-declarations]
 1014 |         if (event->orientation() == Qt::Vertical) {
      |                                ^
In file included from /usr/include/qt/QtGui/QNativeGestureEvent:1,
                 from /home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.h:35,
                 from /home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.cpp:23:
/usr/include/qt/QtGui/qevent.h:222:28: note: declared here
  222 |     inline Qt::Orientation orientation() const { return qt4O; }
      |                            ^~~~~~~~~~~
/home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.cpp:1016:41: warning: 'int QWheelEvent::x() const' is deprecated: Use position() [-Wdeprecated-declarations]
 1016 |             const int offset = event->x();
      |                                         ^
In file included from /usr/include/qt/QtGui/QNativeGestureEvent:1,
                 from /home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.h:35,
                 from /home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.cpp:23:
/usr/include/qt/QtGui/qevent.h:229:16: note: declared here
  229 |     inline int x() const { return int(p.x()); }
      |                ^
/home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.cpp:1036:37: warning: 'int QWheelEvent::delta() const' is deprecated: Use angleDelta() [-Wdeprecated-declarations]
 1036 |             _view.zoom(event->delta() / 80, offset);
      |                                     ^
In file included from /usr/include/qt/QtGui/QNativeGestureEvent:1,
                 from /home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.h:35,
                 from /home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.cpp:23:
/usr/include/qt/QtGui/qevent.h:219:16: note: declared here
  219 |     inline int delta() const  { return qt4D; }
      |                ^~~~~
/home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.cpp:1038:39: warning: 'Qt::Orientation QWheelEvent::orientation() const' is deprecated: Use angleDelta() [-Wdeprecated-declarations]
 1038 |         } else if (event->orientation() == Qt::Horizontal) {
      |                                       ^
In file included from /usr/include/qt/QtGui/QNativeGestureEvent:1,
                 from /home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.h:35,
                 from /home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.cpp:23:
/usr/include/qt/QtGui/qevent.h:222:28: note: declared here
  222 |     inline Qt::Orientation orientation() const { return qt4O; }
      |                            ^~~~~~~~~~~
/home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.cpp:1042:70: warning: 'int QWheelEvent::delta() const' is deprecated: Use angleDelta() [-Wdeprecated-declarations]
 1042 |                                        _view.offset() - event->delta());
      |                                                                      ^
In file included from /usr/include/qt/QtGui/QNativeGestureEvent:1,
                 from /home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.h:35,
                 from /home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.cpp:23:
/usr/include/qt/QtGui/qevent.h:219:16: note: declared here
  219 |     inline int delta() const  { return qt4D; }
      |                ^~~~~
/home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.cpp: In member function 'void pv::view::Viewport::paintMeasure(QPainter&, QColor, QColor)':
/home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.cpp:1539:50: warning: 'int QFontMetrics::width(const QString&, int) const' is deprecated: Use QFontMetrics::horizontalAdvance [-Wdeprecated-declarations]
 1539 |             const int rectW = fm.width(delta_text) + 60;
      |                                                  ^
In file included from /usr/include/qt/QtWidgets/qwidget.h:50,
                 from /usr/include/qt/QtWidgets/QWidget:1,
                 from /home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.h:34,
                 from /home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.cpp:23:
/usr/include/qt/QtGui/qfontmetrics.h:106:9: note: declared here
  106 |     int width(const QString &, int len = -1) const;
      |         ^~~~~
/home/jk/.cache/yay/dsview/src/DSView-1.12/DSView/pv/view/viewport.cpp:1555:38: error: variable 'QPainterPath path' has initializer but incomplete type
 1555 |             QPainterPath path(QPoint(_cur_preX, _cur_preY));
      |                                      ^~~~~~~~~
make[2]: *** [CMakeFiles/DSView.dir/build.make:826: CMakeFiles/DSView.dir/pv/view/viewport.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:96: CMakeFiles/DSView.dir/all] Error 2
make: *** [Makefile:172: all] Error 2
```
It works fine with Qt 5.14.x (or lower).

--- a/DSView/pv/view/viewport.cpp
+++ b/DSView/pv/view/viewport.cpp
@@ -37,7 +37,7 @@
 
 #include <QMouseEvent>
 #include <QStyleOption>
-
+#include <QPainterPath>
 
 #include <math.h>
 

